python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -111,6 +111,12 @@ class RandomForestRegressor(BaseRandomForestModel, DelayedPredictionMixin,
     seed : int (default = None)
         Base seed for the random number generator. Unseeded by default. Does
         not currently fully guarantee the exact same results.
+    ignore_empty_partitions: Boolean (default = False)
+        Specify behavior when a worker does not recieve any data
+        while splitting. When True, it returns the results from workers
+        with data (estimators will be less than n_estimators)
```

Comment:
expand a little "(the number of trained estimators will be less than n_estimators)"